MusicControlActions/src/org/jjazz/ui/musiccontrolactions/ControlToolbarPanel.java

```diff
@@ -290,11 +290,13 @@ private void initComponents()
         add(fbtn_PlaybackPoint);
         add(filler5);
 
+        panel_Tempo.setOpaque(false);
         panel_Tempo.setLayout(new javax.swing.BoxLayout(panel_Tempo, javax.swing.BoxLayout.Y_AXIS));
         panel_Tempo.add(filler6);
 
         spn_Tempo.setModel(new javax.swing.SpinnerNumberModel(120, 30, 350, 1));
         spn_Tempo.setToolTipText(org.openide.util.NbBundle.getMessage(ControlToolbarPanel.class, "ControlToolbarPanel.spn_Tempo.toolTipText")); // NOI18N
+        spn_Tempo.setMaximumSize(new java.awt.Dimension(69, 26));
```

Comment:
I'm not at home so not possible to test it on Mac. I guess it works but I'm a bit concerned of having a hard-coded max dimension, imagine if JSpinner's font is changed to a much larger font...  I think a better way would be to override getMaximumSize() of this particular JSpinner to return getPreferredSize().

Reply:
You are right. I guess I tried this too hastily - as maximum size is used for Fillers, I assumed it would be ok to use it for Spinner too.
I guess I will close this and try to come up with better request...